src/components/value-list-item/value-list-item.tsx

```diff
@@ -189,6 +191,10 @@ export class ValueListItem implements ConditionalSlotComponent, InteractiveCompo
     this.selected = event.detail.selected;
   };
 
+  handleFocus = (event: FocusEvent): void => {
+    this.calciteInternalListItemDragHandleFocused.emit(event);
```

Comment:
Do we need to emit an event nested inside another event? We may want to discuss this pattern if so.